2.6_testers2.ipynb

```diff
@@ -413,10 +442,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our test looks pretty much the same as the earlier Queue tests. But there's more going on\n",
-    "because the computation take multiple cycles so the input enqueue process is blocked as each\n",
-    "GCD is computed. The good news is that test side of this is simple and consistent across \n",
-    "different Decoupled circuits."
+    "Our test looks pretty much the same as the earlier Queue tests.\n",
+    "But there's more going on because the computation take multiple cycles so the input enqueue process is blocked as each GCD is computed.\n",
+    "The good news is that test side of this is simple and consistent across different Decoupled circuits.\n",
+    "\n",
+    "Also introduced here is the new Chisel3 `Bundle` literal notation. Consider the line\n",
```

Comment:
We may actually want to introduce this earlier in the bootcamp

Reply:
Prefer not to take this on at this point

Reply:
Sure, that's fine.